### PR TITLE
NorthStar CT Importer Improvements

### PR DESCRIPTION
**Pull request by imikejackson**
* Allow a subvolume to be imported
* Show user the original volume and subvolume information
* Show user all the .nsidat files that will be read.
* Speed up imports by reading a row at a time instead of a byte at at a time.

Signed-off-by: Michael Jackson <mike.jackson@bluequartz.net>

### CODE REVIEW

**Comment by imikejackson** [comment]
Still need to update the documentation and add a unit test (Somehow)

File: DREAM3DReviewFilters/ReadBinaryCTNorthStar.cpp [hunk 1]
```diff
@@ -206,69 +255,91 @@ int32_t ReadBinaryCTNorthStar::sanityCheckFileSizeVersusAllocatedSize(size_t all
...
-          // Check for progress
-          if(counter > prog)
-          {
-            progressInt = static_cast<int64_t>((static_cast<float>(counter) / totalPoints) * 100.0f);
-            QString ss = QObject::tr("Reading Data || Data File: %1 of %2 || %3% Completed").arg(iter + 1).arg(m_InputFiles.size()).arg(progressInt);
-            notifyStatusMessage(ss);
-            prog = prog + progIncrement;
-          }
-          counter++;
+        index = (importedDims[0] * importedDims[1] * (z - m_StartVoxelCoord[2])) + (importedDims[0] * (y - m_StartVoxelCoord[1])) + (0);
```
Comment (sdonegan) [inline DREAM3DReviewFilters/ReadBinaryCTNorthStar.cpp:328]: Only major comment is here: the data are stored somewhat strangely in the dat files (I think YXZ or something to that effect), so this line is important to get the proper index.  Just wanted to check that this works for either use case of importing the whole volume or a subvolume.  Does this end up being the same equation as the original when importing a full volume?  

This was a really confusing aspect of the filter when we were writing it, so we essentially just checked based on whether the visualization looked correct.

**Review by sdonegan — COMMENTED** [review]
Generally looks good, definitely useful functionality.  Just want a double-check on the index computation.  I generally just checked by reading in a known file and making sure the data weren't garbage.

File: DREAM3DReviewFilters/ReadBinaryCTNorthStar.cpp (hunk 1 above)
Comment (imikejackson) [reply DREAM3DReviewFilters/ReadBinaryCTNorthStar.cpp:328]: I checked visualizations of both full volumes and sub-volumes and both appeared correct.

**Pull request merged**